pyproject.toml

```diff
@@ -1,14 +1,27 @@
 [build-system]
-requires = ["setuptools>=42", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["setuptools>=42", "wheel", "setuptools-git-versioning>=2.0,<3"]
+build-backend = "setuptools.build_meta:__legacy__"
+
+[tool.setuptools-git-versioning]
+count_commits_from_version_file = true
+enabled = true
+version_file = "src/mrpro/VERSION"
+template = "{tag}{env:MRPROVERSIONSUFFIX:''}"
+dev_template = "{tag}{env:MRPROVERSIONSUFFIX:''}"
+dirty_template = "{tag}+dirty{sha}"
+
+[tool.setuptools]
+include-package-data = true
+
+[tool.setuptools.package-data]
+"mrpro" = ["VERSION"]
```

Comment:
LICENSE ?

Reply:
According to https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license listing the license in the classifiers is enough, or did I get that wrong?

Reply:
both of us are wrong: LICENSE should be automatically included by setuptools when building the distribution.